test/Interop/SwiftToCxx/unsupported/unsupported-decls-in-cxx.swift

```diff
@@ -10,9 +10,49 @@
 // CHECK-NOT: Unsupported
 // CHECK: supported
 
+import Distributed
+
 public func supported() {}
 
-@_expose(Cxx) // expected-error {{actor 'ActorClassUnsupported' can not be exposed to C++}}
-public actor ActorClassUnsupported {
-    public func test() {}
+@_expose(Cxx)
+public actor ActorClass {
+    @_expose(Cxx) // expected-error {{actor-isolated instance method 'unsupported()' can not be exposed to C++}}
+    public func unsupported() {}
+
+    @_expose(Cxx) // expected-error {{actor-isolated property 'prop' can not be exposed to C++}}
+    public var prop: Int { 42 }
+
+    @_expose(Cxx) // ok
+    public init() {
+    }
+
+    @_expose(Cxx) // ok
+    nonisolated public func supported() {}
+
+    @_expose(Cxx) // ok
+    nonisolated public var prop2: Int { 42 }
+}
+
+@_expose(Cxx)
+public distributed actor DistributedActorClass {
+    public typealias ActorSystem = LocalTestingDistributedActorSystem
+
+    @_expose(Cxx) // expected-error {{actor-isolated instance method 'unsupported()' can not be exposed to C++}}
+    public func unsupported() {}
+
+    @_expose(Cxx) // expected-error {{actor-isolated property 'prop' can not be exposed to C++}}
+    public var prop: Int { 42 }
+
+    @_expose(Cxx) // expected-error {{actor-isolated distributed instance method 'dist()' can not be exposed to C++}}
+    distributed public func dist() {}
+
+    @_expose(Cxx) // ok
+    public init() {
```

Comment:
```suggestion
    public init(actorSystem: LocalTestingDistributedActorSystem) {
        self.actorSystem = actorSystem
```

is what you'd need to do to make sure this is a 100% correct actor; this should have failed to compile tbh as it is declared now - but the other errors probably prevented it from going into those checks

Reply:
Sounds good, I'll commit this as a follow-up!